amps/ags/rm-community/rm-community-repo/config/alfresco/module/org_alfresco_module_rm/alfresco-global.properties

```diff
@@ -139,3 +139,21 @@ content.metadata.async.extract.6.enabled=false
 
 # Max number of entries returned in Record search view
 rm.recordSearch.maxItems=500
+
+#
+# Hold bulk
+#
+# The number of worker threads.
+rm.hold.bulk.threadCount=2
+# The maximum number of total items to process in a single bulk operation.
+rm.hold.bulk.maxItems=1000
+# The number of entries to be fetched from the Search Service as a next set of work object to process.
+rm.hold.bulk.batchSize=100
+# The number of entries to process before reporting progress.
+rm.hold.bulk.logging.interval=1000
```

Comment:
Isn't this a bit high given that the default `maxItems` for a bulk operation is `1000` as well? I assume this means we would get progress information only after the whole operation has completed, which seems a bit extreme?

Reply:
I agree, I think `100` should be enough.